wgsl/index.bs

```diff
@@ -2688,6 +2688,38 @@ the contents of the existing table.
   <tr><td>is_normal(float) -&gt; bool<td>OpIsNormal
 </table>
 
+## Integer built-in functions ## {#integer-builtin-functions}
+
+<table class='data'>
+  <thead>
+    <tr><td>Precondition<td>Built-in<td>Description
+  </thead>
+  <tr algorithm="scalar popcount"><td>
```

Comment:
I find `popcount` confusing, and also - slightly inconsistent (i.e. we have "reverse_bits" with underscore between words but not "population_count" or something like this). Would it be possible to consider other names, maybe "count_bits()` or something similar?

Reply:
count_ones is a name I like, given other names like count_leading_zeros in other areas.

Reply:
GLSL has `bitCount`

I like countOneBits, which goes well with reverseBits. I'll change to that.